content/src/fieldConfig/business-formation.json

```diff
@@ -26,7 +26,9 @@
       "viewMoreButtonText": "View more",
       "amendmentInfo": "#### Review your information before submitting to the New Jersey Division of Revenue and Enterprise Services.\n If you need to make a change after submitting, you must file an [amendment online.](https://www.njportal.com/dor/businessamendments)",
       "viewLessButtonText": "View less",
-      "genericErrorText": "This field requires information."
+      "genericErrorText": "This field requires information.",
+      "readMoreText": "Read More",
+      "readLessText": "Read Less"
```

Comment:
[pebble] We already have "viewMoreButtonText" and "viewLessButtonText" right here. Are you sure this isn't a design oversight? I can't imagine they sometimes want some fields to say "View More" and some to say "Read More" inconsistently. I think this needs to be confirmed with design and/or content and then (assuming they want it all to be one consistent copy) remove the addition. 

Bear in mind that the whole point of the JSON structure is that the keys don't have to change when the values do. So if they want it to say "Read More" everywhere, we should continue to use the existing "viewMoreButtonText" and change its value rather than adding a new key just because the text has changed.